content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,259 @@
+
+.. image:: avalara_portal/avalara_transactions.png
+   :align: center
+   :alt: Avalara portal with the transactions shortcut highlighted.
+
+Edit transaction
+----------------
+
+Click into a transaction to reveal more details about the transaction. These details include
+sections on :guilabel:`Invoice detail`, :guilabel:`Additional info`, and :guilabel:`Customer info`.
+Click :icon:`fa-pencil` :guilabel:`Edit document details` to make changes to the transaction.
+
+A :guilabel:`Discount` can be added to adjust the invoice. This is especially useful in cases where
+the transaction has already synced with Avalara / *AvaTax*, and changes need to be made afterward.
+
+.. _avalara_portal/filter:
+
+Filter
+------
+
+Filter transactions on the :guilabel:`Transactions` page, by setting the :guilabel:`From` and
+:guilabel:`To` fields, and configuring other fields to filter by, including:
+
+ - :guilabel:`Document Status`: any of the following options, :guilabel:`All`, :guilabel:`Voided`,
+   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Document Code` : any of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Customer/Vendor Code`: the customer/vendor code in Odoo (e.g. `Contact18`).
+ - :guilabel:`Country`: the country this tax was calculated in; this is a text field.
+ - :guilabel:`Region`: the region of the country, which varies based on the :guilabel:`Country`
+   selection.
+
+Click :icon:`fa-plus` :guilabel:`Filters` to access the following filter conditions:
+
+ - :guilabel:`Document Type`: any of the following selections, :guilabel:`All`, :guilabel:`Sales
+   Invoice`, :guilabel:`Purchase Invoice`, :guilabel:`Return Invoice`, :guilabel:`Inventory Transfer
+   Inbound Invoice`, :guilabel:`Inventory Transfer Outbound Invoice`, or :guilabel:`Customs
+   Invoice`.
+ - :guilabel:`Import ID`: represents the import ID of the document.
+
+Sort by
+-------
+
+On the :guilabel:`Transactions` page, transactions will be listed below, according to the set
+:ref:`avalara_portal/filter`, located in the top half of the page. The following columns are
+available by default, to sort by ascending or descending order:
+
+ - :guilabel:`Doc Code` : either of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Doc Status` : either of the following options, :guilabel:`All`, :guilabel:`Voided`,
+   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Cust/Vendor Code` : this is the customer/vendor code in Odoo (e.g. Contact18).
+ - :guilabel:`Region` : this is the region of the country, this will vary based on the
+   :guilabel:`Country` selection.
+ - :guilabel:`Amount` : the numeric amount of the total amount on the Odoo document.
+ - :guilabel:`Tax` : the numeric amount of the tax applied to the total.
```

Comment:
```suggestion
- :guilabel:`Doc Code`: either of the following options, :guilabel:`Exactly match`,
  :guilabel:`Starts with`, or :guilabel:`Contains`.
- :guilabel:`Doc Status`: either of the following options, :guilabel:`All`, :guilabel:`Voided`,
  :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
- :guilabel:`Cust/Vendor Code`: this is the customer/vendor code in Odoo (e.g. Contact18).
- :guilabel:`Region`: this is the region of the country, this will vary based on the
  :guilabel:`Country` selection.
- :guilabel:`Amount`: the numeric amount of the total amount on the Odoo document.
- :guilabel:`Tax`: the numeric amount of the tax applied to the total.
```